health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/ViewHierarchy.js

```
@@ -0,0 +1,440 @@
+
+            });
+            setDataCreateToast(false);
+            setShowToast({ label: `${t("WBH_HIERARCHY_CREATED")}` });
+            return res;
+        }
+        catch(resp){
+            // console.log("error raised", resp);
+            let label = `${t("WBH_BOUNDARY_CREATION_FAIL")}: `;
+            resp?.response?.data?.Errors?.map((err, idx) => {
+            if (idx === resp?.response?.data?.Errors?.length - 1) {
+                label = label + t(Digit.Utils.locale.getTransformedLocale(err?.code)) + ".";
```

Biome

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)